napari/layers/points/points.py

```diff
@@ -390,6 +390,8 @@ def __init__(
         else:
             self._current_edge_color = self.edge_color[-1]
             self._current_face_color = self.face_color[-1]
+            self._face_color = []
+            self._edge_color = []
```

Comment:
Even though it's empty, I think it might be better to initialize as a numpy since that is the assumed type for the colors.

```suggestion
            self._face_color = np.empty((0, 4))
            self._edge_color = np.empty((0, 4))
```